source/markdown/posts/1.md

```diff
@@ -0,0 +1,5 @@
+#### Hive NYC Partnership
+
+Hive NYC partnered with the NYC City Council to present the Digital Inclusion Summit on August 16, 2016. Together, community members, technologists, non-profit organizations, and members of the New York City Council for conversations about the future of voter engagement, Participatory Budgeting, and other public initiatives.
```

Comment:
Hive NYC partnered with New York's City Council to present the Digital Inclusion Summit on August 16, 2016 to bring together stakeholders in a conversation about the future of voter engagement, female activism, Participatory Budgeting, and more.